src/hashtable.py

```diff
@@ -0,0 +1,26 @@
+class HashTable(object):
+
+    def __init__(self, buckets=1024):
+        self.buckets = buckets
+        self.init_table()
+
+    def init_table(self):
+        self._table = []
+        for x in range(self.buckets):
+            self._table.append([])
+
+    def get(self, key):
+        bn = self._hash(key) % self.buckets
+        for k, value in self._table[bn]:
+            if k == key:
+                return value
+
+    def set(self, key, value):
```

Comment:
I suppose you're letting Python do the heavy lifting here when enforcing that your input key should be a string, but I feel like it'd be better if the `TypeError` raised when a non-string is given explicitly referenced that necessity.